Symfony/CS/Tests/Tokenizer/TokensTest.php

```diff
@@ -817,7 +817,7 @@ public function testShortOpenTagMonolithicPhpDetection($source, $monolitic)
          * short_open_tag setting is ignored by HHVM
          * @see https://github.com/facebook/hhvm/issues/4758
          */
-        if (!ini_get('short_open_tag') && !defined('HHVM_VERSION')) {
+        if (!defined('HHVM_VERSION') && !ini_get('short_open_tag')) {
```

Comment:
Why are these changed? This doesn't make sense. You are making average execution slower because HHVM_VERSION is always not defined during most executions.